cmd/kyma/provision/minikube/cmd.go

```diff
@@ -58,11 +58,12 @@ func NewCmd(o *Options) *cobra.Command {
 	}
 
 	cmd := &cobra.Command{
-		Use:     "minikube",
-		Short:   "Provisions Minikube.",
-		Long:    `Use this command to provision a Minikube cluster for Kyma installation. It requires to have Minikube installed upfront, see also https://github.com/kubernetes/minikube`,
-		RunE:    func(_ *cobra.Command, _ []string) error { return c.Run() },
-		Aliases: []string{"m"},
+		Use:        "minikube",
+		Short:      "Provisions Minikube.",
```

Comment:
Just curious.. how long are we going to support Minikube and are the automated tests also covering Minikube + new installation approach?

Reply:
with minikube one needs to use the old way of installation.